elevate-project/controllers/project.js

```diff
@@ -143,6 +143,75 @@ const readUser = async (req, res, selectedConfig) => {
 		return res.status(500).json({ error: 'Internal Server Error' })
 	}
 }
+const readUserTitle = async (req,res,selectedConfig)=>{
+	try {
+
+		
+		if(selectedConfig.service){
+			req['baseUrl'] = process.env[`${selectedConfig.service.toUpperCase()}_SERVICE_BASE_URL`]
+
+		}		
+		const parameterisedRoute = req.params.id ? selectedConfig.targetRoute.path.replace('/:id', `/${req.params.id}`) : selectedConfig.targetRoute.path;
+		let headers
+	
+		if (req.params.id) {
+			headers = {
+			'X-auth-token': req.headers['x-auth-token'],
+			'Content-Type': 'application/json',
+			}
+		}		
+
+		let response = await requesters.get(req.baseUrl, parameterisedRoute, headers)
```

Comment:
@MallanagoudaB what happens if you get failure response from entity service, handle that